docs/layers/data/data.mdx

```diff
@@ -1,7 +1,64 @@
+
+    <PrimaryCTA to="/components/library/aws/aurora-postgres/">Get Started</PrimaryCTA>
+  </TabItem>
+
+  <TabItem value="aurora-mysql" label="Aurora MySQL">
+    Aurora MySQL is a fully managed, MySQL-compatible relational database service that combines the performance and availability of high-end commercial databases with the simplicity and cost-effectiveness of open-source databases. Aurora MySQL is a good choice for applications that require the features and capabilities of MySQL with the scalability, performance, and reliability of a managed service.
+
+    <PrimaryCTA to="/components/library/aws/aurora-mysql">Get Started</PrimaryCTA>
+  </TabItem>
+
+  <TabItem value="rds" label="RDS">
+    Amazon RDS is a managed relational database service that supports multiple database engines, including MySQL, PostgreSQL, MariaDB, Oracle, and SQL Server. RDS is a good choice for applications that require the features and capabilities of a relational database with the scalability, performance, and reliability of a managed service.
+
+    <PrimaryCTA to="/components/library/aws/rds">Get Started</PrimaryCTA>
+  </TabItem>
+
+  <TabItem value="redshift" label="RedShift">
+    Amazon Redshift is a fully managed, petabyte-scale data warehouse service that provides fast query performance using SQL and business intelligence tools. Redshift is a good choice for applications that require high-performance analytics and data warehousing with built-in security, backup, and restore capabilities.
+
+    <PrimaryCTA to="/components/library/aws/redshift">Get Started</PrimaryCTA>
+  </TabItem>
+</Tabs>
+
+### NoSQL Database Options
+
+The reference architecture also supports several NoSQL database options, including DynamoDB, DocumentDB, and more. SQL and/or NoSQL databases can always be deployed side by side to support different use cases.
+
+<Tabs queryString="nosql">
+  <TabItem value="dynamodb" label="DynamoDB">
+    DynamoDB is a fully managed NoSQL database service that provides fast and predictable performance with seamless scalability. DynamoDB is a good choice for applications that require low-latency, high-throughput access to data with built-in security, backup, and restore capabilities.
+
+    <PrimaryCTA to="/components/library/aws/dynamodb">Get Started</PrimaryCTA>
+  </TabItem>
+
+  <TabItem value="documentdb" label="DocumentDB">
+    Amazon DocumentDB is a fully managed, MongoDB-compatible document database service that provides fast and scalable performance with built-in security, backup, and restore capabilities. DocumentDB is a good choice for applications that require the features and capabilities of MongoDB with the scalability, performance, and reliability of a managed service.
+
+    <PrimaryCTA to="/components/library/aws/documentdb">Get Started</PrimaryCTA>
+  </TabItem>
```

Comment:
Let's add Elasticache Redis as a very common option.

Reply:
And Elasticsearch

Reply:
added